SlimApp/HomeView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ContentView.swift
+//  SlimApp
+//
+//  Created by Aleksandr Shapovalov on 10/01/23.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    var body: some View {
+        ZStack {
+            Color("Background")
+                .ignoresSafeArea()
+            NavigationView {
+                ScrollView {
+                    content
+                        
+                }
+
+                .navigationTitle("Clients")
+                .toolbar {
+                    Button {
+                        //
+                    } label: {
+                        Image(systemName: "plus")
+                    }
+                }
+            }
+        }
+    }
+}
+
+
+extension HomeView {
+    var content: some View {
+        VStack(alignment: .leading, spacing: 0) {
+
+//            Text("Clients")
+//                .font(.system(size: 34, weight: .bold))
+//                .frame(maxWidth: .infinity, alignment: .leading)
+//                .padding(.horizontal, 20)
+
+
+//            Divider()
```

Comment:
Is this code still going to be used or can it be discarded?

Reply:
fixed on another branch with ClientList